__docs__/wonder-blocks-search-field/search-field.argtypes.ts

```diff
@@ -1,6 +1,5 @@
 export default {
     clearAriaLabel: {
-        description: `ARIA label for the clear button. Defaults to "Clear search".`,
```

Comment:
praise: Thanks for cleaning this up!

I will also start removing descriptions from story argtypes as I go along so we can rely on the prop docs to avoid duplication 😄